src/extension/packagerStatusIndicator.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {window, Disposable, StatusBarItem, StatusBarAlignment} from "vscode";
+
+/**
+ * Updates the Status bar with the status of React Native Packager.
+ */
+
+export enum PackagerStatus {
+    PACKAGER_STARTED = 0,
+    PACKAGER_STOPPED
+}
+
+export class PackagerStatusIndicator implements Disposable {
+    private packagerStatusItem: StatusBarItem;
+    private static PACKAGER_STARTED_STATUS_STR: string = "React Native Packager: Running";
```

Comment:
I think we should change "Running" for "Started". Which the current implementation if the packager dies or something, it'll still say Running. Started seems to be slightly more truthful in that we started it, but it doesn't imply so strongly what happened after that.